igannotator/output/mae.py

```diff
@@ -65,22 +66,15 @@ def write_mae_representation(
             "Deontic": "D",
         }
         offset = 0
-
-        all_tags = defaultdict(list)
-
+        id = 0
         for sentence, tags in sentences_with_tags:
             for tag_name, start, stop, tag_text in tags:
                 spans = f"{offset + start}~{offset + stop}"
                 tag_repr = f'<{tag_name} id="{tag_to_shorthand[tag_name]}{tag_ids[tag_name]}" spans="{spans}" text="{sentence[start:stop]}" />\n'
-                all_tags[tag_name].append(tag_repr)
+                output.write(tag_repr)
```

Comment:
@rzepinskip is this change really intended?

Reply:
I suppose I unintentionally merged some reverted code. Please fix accordingly in new PR.